src/lib/verifier/prod.ml

```diff
@@ -191,7 +189,7 @@ module Worker_state = struct
                result
 
              let get_blockchain_verification_key () =
-               Lazy.force B.Proof.verification_key
+               Deferred.return blockchain_verification_key
```

Comment:
Let's remove `get_blockchain_verification_key` from verifier and its interface, it's never used